website/blog/2022-06-16-resources-migrating-your-react-native-library-to-the-new-architecture.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Resources - Migrating Your React Native Library to the New Architecture
```

Comment:
```suggestion
title: Helping migrate React Native libraries to the New Architecture
```
this title is more aligned with the tone, also I'm not a big fan of the "Resources -" at the start, feels like it should almost be a tag instead